cohere/__init__.py

```diff
@@ -8,6 +8,7 @@
 EMBED_URL = 'embed'
 CLASSIFY_URL = 'classify'
 EXTRACT_URL = 'extract'
+TRANSCRIBE_URL = 'whisper/transcribe'
```

Comment:
you can use 'transcribe'
```suggestion
TRANSCRIBE_URL = 'transcribe'
```

Reply:
Done :)